### PR TITLE
cray power monitoring module

### PR DESCRIPTION
**Pull request by tallendev**

### CODE REVIEW

File: src/report.c [hunk 1]
```diff
@@ -193,59 +198,66 @@ void compute_local_region_stats(region_t *reg, regstats_t *stats, int incl, int
   wallt = reg->wtime;
 
 #ifdef HAVE_PAPI
-  gflops = ipm_papi_gflops(reg->ctr, wallt);
+  gflops=0.0;//gflops = ipm_papi_gflops(reg->ctr, wallt);
```
Comment (cdaley) [inline src/report.c]: Is there a reason we need to remove ipm_papi_gflops?

File: src/report_xml.c [hunk 2]
```diff
@@ -248,20 +250,21 @@ int xml_perf(void *ptr, taskdata_t *t) {
...
   assert(t->rstack->child);
   reg = t->rstack->child;  */ /* this is ipm_main */
 
   reg = &ipm_app;
-  
+
   procmem = task.procmem;
 #ifdef HAVE_PAPI
-  gflops = ipm_papi_gflops(reg->ctr, reg->wtime);
+  gflops=0.0;
```
Comment (cdaley) [inline src/report_xml.c]: As before. Why remove ipm_papi_gflops? Why add unistd.h?

File: test/test.pmon/README [hunk 3]
```diff
@@ -0,0 +1,4 @@
+short test demonstrating power management over sleeps. Sleeps are 5, 10, 15 
+seconds. the most deeply-nested regions should have energy scaled to the length
+of the sleep. Outer regions should be the sum of the inner regions. Power consumption
+is low because of the sleeps.
```
Comment (cdaley) [inline test/test.pmon/README]: Can you add example IPM output from test.pmon to the README file highlighting which values should be 2x/3x larger?

File: src/report_xml.c [hunk 4]
```diff
@@ -398,30 +401,43 @@ int xml_internal(void *ptr, taskdata_t *t) {
...
+  double gflops=0.0;
+
+
   nc=0;
   for( i=0; i<MAXNUM_PAPI_EVENTS; i++ ) {
     if( (papi_events[i].name[0]) )
       nc++;
   }
-  
-  gflops = ipm_papi_gflops(reg->ctr, reg->wtime);
```
Comment (cdaley) [inline src/report_xml.c]: As before. Why remove ipm_papi_gflops?

File: test/test.pmon/README (hunk 3 above)
Comment (tallendev) [inline test/test.pmon/README]: Please check revised pull request.

**Comment by cdaley** [comment]
Look good. Thanks for the clarification regarding ipm_papi_gflops. I'm amazed the bug of it being reported on rank 0 only has persisted for so many years. This is a very low priority bug fix because most hardware cannot measure FLOPs today.

**Pull request merged**